src/Client.php

```diff
@@ -69,8 +90,12 @@ public function deleteAnnotation(Annotation $annotation)
         return $this->client->delete($annotation);
     }
 
-    public function search(SearchQuery $query)
+    public function search(SearchQuery $query, $asResponse = false)
```

Comment:
Not a fan of the return type changing depending on the parameters you give it.

Reply:
No me neither, but I wanted to preserve the BC compatilbility with other modules currently using it to search expecting JSON back. Definitely something for the back log though